packages/strapi-plugin-content-type-builder/admin/src/translations/en.json

```diff
@@ -171,6 +171,9 @@
   "popUpWarning.bodyMessage.category.delete": "Are you sure you want to delete this category? All the components will also be deleted.",
   "popUpWarning.bodyMessage.component.delete": "Are you sure you want to delete this component?",
   "popUpWarning.bodyMessage.contentType.delete": "Are you sure you want to delete this collection type?",
+  "popUpWarning.draft-publish.button.confirm": "Yes, disable",
+  "popUpWarning.draft-publish.message": "If you disable the Draft/Publish system, your previously created drafts will be deleted?",
```

Comment:
is it intented to have the question mark here?

Reply:
Why is it `your previously created drafts` and not just `your drafts for this collection-type`?

Reply:
@JabStrapi

Reply:
We go `your drafts will be deleted.`